implementations/rust/ockam/ockam_abac/Cargo.toml

```diff
@@ -41,24 +49,38 @@ minicbor = { version = "0.24.1", features = ["derive", "alloc"] }
 ockam_core = { version = "0.112.0", path = "../ockam_core", default-features = false }
 ockam_identity = { version = "0.116.0", path = "../ockam_identity", default-features = false }
 ockam_node = { version = "0.121.0", path = "../ockam_node", default-features = false }
-once_cell = { version = "1.19.0", default-features = false, features = ["alloc"] }
+once_cell = { version = "1.19.0", default-features = false, features = [
```

Comment:
Can you please keep the default formatting for `features`?

Reply:
Done